tobac/tracking.py

```diff
@@ -178,6 +178,11 @@ def linking_trackpy(
     #    from trackpy import filter_stubs
     #    from .utils import add_coordinates
 
+    if (v_max is None) and (d_min is None) and (d_max is None):
```

Comment:
Now that I look again at the explanation of `d_min` in the docstring, I think we should exclude it here, since its original purpose is not to define a search range (The current implementation, however, only mimics `d_max`).

Since the parameter will eventually be deprecated, it is probably better not to promote its use in an error.

Reply:
I agree with your thoughts here, Nils. I'll wait to review until this is changed. I'd say that we should leave`d_min` in the if statement here, but remove it in the error message.

Reply:
I think you are both absolutely right. I have updated the error message accordingly.